glide.yaml

```diff
@@ -55,3 +55,7 @@ import:
   subpackages:
   - unix
 - package: github.com/samalba/dockerclient
+- package: github.com/samuel/go-zookeeper
+  version: 8c6a1d79a44d2742da3660540d1fd43f4ddcfc8c
+  subpackages:
+    - zk
```

Comment:
Not clear what should I do here. IIRC, `glider update` doesn't change anything to commit on my repo. Or are you referring to the Make file?

> Il giorno 12 set 2016, alle ore 16:16, Tim Gross <notifications@github.com> ha scritto:
> 
> In glide.yaml:
> 
> > @@ -55,3 +55,7 @@ import:
> >    subpackages:
> >    - unix
> >  - package: github.com/samalba/dockerclient
> > +- package: github.com/samuel/go-zookeeper
> > +  version: 8c6a1d79a44d2742da3660540d1fd43f4ddcfc8c
> > +  subpackages:
> > +    - zk
> @giorgio-v you'll still need to do this to get Travis to pass.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>